src/platforms/python/common/integrations/celery/index.mdx

```diff
@@ -3,7 +3,6 @@ title: Celery
 redirect_from:
   - /clients/python/integrations/celery/
   - /platforms/python/celery/
-  - /platforms/python/guides/celery/hints/
```

Comment:
This would not resolve anymore, would redirect to `/platforms/python/hints/` (HTTP 404)

Reply:
fix in vercel.json

Reply:
checked. works.